### ISSUE TITLE
Missing find_package(TINYXML2)

### ISSUE THREAD

**Issue by traversaro** [ignitionrobotics/ign-gazebo#700]
I may be wrong, but it seems that TINYXML2::TINYXML2 is linked to some libraries (see https://github.com/ignitionrobotics/ign-gazebo/blob/f9a5ecefc6ad068e9b1e50eab2333a49abd748f0/src/gui/plugins/CMakeLists.txt#L37) but it is never found explicitly via `find_package(TINYXML2)` .

**Comment by chapulina** [comment]
Yeah I think it's a good idea to find it explicitly. I think we're just relying on `ign-gui` to find it right now.
